2017-01-31  Jakub Jelinek  <jakub@redhat.com>

	PR target/79170
	* config/rs6000/rs6000.c (expand_block_compare): Temporarily disable.

--- gcc/config/rs6000/rs6000.c.jj	2017-01-31 09:25:59.000000000 +0100
+++ gcc/config/rs6000/rs6000.c	2017-01-31 10:17:21.131929670 +0100
@@ -19462,7 +19462,7 @@ expand_block_compare (rtx operands[])
      with carry instructions do not generate the 64-bit
      carry and so we must emit code to calculate it ourselves.
      We choose not to implement this yet.  */
-  if (TARGET_32BIT && TARGET_POWERPC64)
+  if ((TARGET_32BIT && TARGET_POWERPC64) || 1)
     return false;
 
   /* If this is not a fixed size compare, just call memcmp.  */
